DEVELOP.md

```diff
@@ -0,0 +1,139 @@
+# Development
+
+This document will guide you through how to
+. Set up the local development environment
+. Build web UI and configure backend service
+. Run the cloud gaming demo for the local development
+. Add additional games
+
+## Web UI
+
+The web UI for the clouding gaming demo is built with [Flutter](https://flutter.dev/).
+To set up the development environment and build the UI, make sure that you have Flutter installed on your machine.
+Follow the [installation instructions](https://docs.flutter.dev/get-started/install/linux) to install it.
+
+### How to build
+
+To build the UI, simply run:
+
+  ```
+  cd ui && flutter build web
+  ```
+This builds the UI and places outputs in the `ui/build/web` directory.
+Refer to the official [documentation](https://docs.flutter.dev/get-started/web) for details on how to build web apps with Flutter.
+
+### How to deploy
+
+Create a `static` folder under the `backend/service` directory and copy everything that is in the `ui/build/web` directory into it.
+
+  ```
+  mkdir -p backend/service/static
+  cp -av ui/build/web/* backend/service/static
+  ```
+Those static resources are then served by backend service when you access the web UI.
+
+
+## Backend
+
+The backend service is not just for serving the web UI, but also interacts with Anbox Stream Gateway and provides a set of restful APIs for clients(web UI) to perform operations like list games or launch a game.
+
+### Install the dependency
```

Comment:
```suggestion
### Install the dependencies
```
I understand it's only one, but it sounds much better like this imo.

Reply:
+1

Reply:
Fixed